assets/src/components/v2/elevator/closures.tsx

```diff
@@ -161,6 +162,24 @@ const UpcomingClosure = ({
   );
 };
 
+const NoCurrentClosures = () => {
+  return (
+    <div className="no-closures">
+      <NormalServiceIcon height={150} width={150} fill="#145A06" />
+      <div className="no-closures-header">All MBTA elevators are working.</div>
```

Comment:
nit: We've been (somewhat haphazardly) following a BEM-style naming convention for classes, where discrete elements of a single component are named as e.g. `no-closures__header` (`no-closures` is the "block", `header` is the "element"). This makes it a little easier to understand the nesting of elements in cases where they have a fixed nesting. If an SCSS selector for an element is nested inside the selector for its block, you can also shorthand to e.g. `&__header`. An example of where this is done already on elevator screens is `.upcoming-closure__title`.

Reply:
cool I'll use that convention. thanks for explaining!